src/DistrictCard.js

```diff
@@ -21,3 +22,22 @@ const DistrictCard = ({ location, data }) => {
 }
 
 export default DistrictCard;
+
+DistrictCard.propTypes = {
```

Comment:
Proptypes are thorough. I like that you checked to make sure that all proptypes for years were numbers since we have a method that should have changed the data to numbers if it was not